fml/synchronization/waitable_event.h

```diff
@@ -52,9 +52,8 @@ class AutoResetWaitableEvent final {
   // called) each |Signal()| unblocks exactly one |Wait()|.
   void Wait();
 
-  // Like |Wait()|, but with a timeout. Also unblocks if |timeout_microseconds|
-  // without being signaled in which case it returns true (otherwise, it returns
-  // false).
+  // Like |Wait()|, but with a timeout. Also unblocks if |timeout| without being
```

Comment:
nit: the second sentence doesn't parse easily. How about `Also unblocks if |timeout| expires without...`?

Reply:
Done.